src/element/bounds.ts

```diff
@@ -535,7 +535,18 @@ export const getResizedElementAbsoluteCoords = (
     bounds = getBoundsFromPoints(points);
   } else {
     // Line
-    const curve = generateLinearElementShape(element);
+    const gen = rough.generator();
+    const curve =
+      element.strokeSharpness === "sharp"
+        ? gen.linearPath(
+            points as [number, number][],
+            generateRoughOptions(element),
+          )
+        : gen.curve(
+            points as [number, number][],
+            generateRoughOptions(element),
+          );
```

Comment:
I just spotted a bug with resizing a single rotated line. Rolling back this change fixed it, but I don't really understand why. There's not much difference between this code and `generateLinearElementShape`